### PR TITLE
Add CountByQueryBuilder() to SLC

### PR DESCRIPTION
**Pull request by TatsuNet**
Add CountByQueryBuilder() method to SLC.
And this can be called from Tx.CountByQueryBuilder() method.

### CODE REVIEW

**Comment by goccy** [comment]
Please add `CountByQueryBuilderContext` API too !

File: rapidash.go [hunk 1]
```diff
@@ -418,10 +418,25 @@ func (tx *Tx) FindByQueryBuilderContext(ctx context.Context, builder *QueryBuild
 }
 
 func (tx *Tx) CountByQueryBuilder(builder *QueryBuilder) (uint64, error) {
+	 count, err := tx.CountByQueryBuilderContext(context.Background(), builder)
+	 if err != nil {
+	 	return 0, err
```
Comment (goccy) [inline rapidash.go]: Please wrap `err` by `xerrors.Errorf(": %w", err)`

Comment (TatsuNet) [reply rapidash.go]: https://github.com/knocknote/rapidash/pull/10/commits/599be7f28348fb0e5c07f71abfd9fe5807569c2d

**Comment by goccy** [comment]
LGTM !

**Pull request merged**